src/public/inventory.ts

```diff
@@ -133,5 +133,32 @@ export const inventoryPublicHandlers: PublicNatsHandler[] = [
         {
             queue: SERVICE_NAME
         }
+    ],
+    [
+        "PUT",
+        "inventory.*",
+        async (subscription: Subscription): Promise<void> => {
+            for await (const message of subscription) {
+                try {
+                    const natsConnection = getConnection();
+                    const { body } = jsonCodec.decode(
+                        message.data
+                    ) as AirlockPayload;
+
+                    const response = await natsConnection.request(
+                        "item-store.transfer_inventory_item",
+                        jsonCodec.encode(body)
```

Comment:
`transfer_inventory_item` requires an `inventory_item_id`, is it supposed to come from the payload or from the URL? Note that the Postman collection doesn't pass an `inventory_item_id`

Reply:
I added `const urlParameter = String(message.subject).split(".")[2];` so that transfer_inventory_item can use inventory_item_id in the payload